glib/src/log.rs

```diff
@@ -581,10 +632,10 @@ macro_rules! g_info {
 /// ```
 #[macro_export]
 macro_rules! g_debug {
-    ($log_domain:expr, $format:expr, $($arg:expr),* $(,)?) => {{
+    ($log_domain:expr, $format:literal, $($arg:expr),* $(,)?) => {{
```

Comment:
The examples in the docs above everywhere still use `Some("test")` as log domain. That should allow omitting the log domain completely, or passing it as `"test"` without the `Option` wrapping :)

Reply:
Oh right, it's clearly not mandatory.

Reply:
Well, not anymore at least.